app-crypt/ccid/files/ccid-1.5.4-remove-flex-configure-dependency.patch

```diff
@@ -0,0 +1,16 @@
+Bug: https://bugs.gentoo.org/221361
+Reverts upstream commit eec7cdf0
+
+--- a/configure.ac
++++ b/configure.ac
+@@ -32,11 +32,6 @@ AC_PROG_CPP
+ AC_PROG_INSTALL
+ AC_PROG_MAKE_SET
+ AC_PROG_LN_S
+-AC_PROG_LEX([noyywrap])
+-if test $LEX = ":"
+-then
+-    AC_MSG_ERROR([no lex or flex found])
+-fi
```

Comment:
Sorry but this actually makes no sense.

The upstream commit in question added the AC_MSG_ERROR, it did NOT add the AC_PROG_LEX. Removing the latter means that now you cannot run automake, which in turn is why you ran `eautoconf` instead of `eautoreconf` in src_prepare. It is generally a problem because if someone later ends up patching the Makefile.am for any reason, you will be unable to make it stick (and then implicit maintainer rules may fire anyways and make everything fail).

I submitted a PR upstream to make this work correctly: https://github.com/LudovicRousseau/CCID/pull/140